repobee_feedback/feedback.py

```diff
@@ -111,6 +112,53 @@ def command(self, api: plug.PlatformAPI):
         callback(self.args, api)
 
 
+class GenerateMultiIssueFile(plug.Plugin, plug.cli.Command):
+    __settings__ = plug.cli.command_settings(
+        help="auto generate multi-issue file",
+        description="Will generate a multi-issue file template "
+        "where each pair of student and assignment (args) passed "
+        "will become an issue that starts with the line "
+        "#ISSUE#<STUDENT_REPO_NAME>#<ISSUE_TITLE>, followed by its "
+        "body. Title and body should be filled appropriately later.",
+        category=plug.cli.CoreCommand.issues,
+        action="generate-multi-issue-files",
+        base_parsers=[plug.BaseParser.ASSIGNMENTS, plug.BaseParser.STUDENTS],
+    )
+
+    assignments = plug.cli.option(required=True)
+    students = plug.cli.option(required=True)
+
+    def command(
+        self,
+    ):
+        _generate_multi_issue_file(self.args.students, self.args.assignments)
```

Comment:
I would suggest making this function return a string instead, as that's easier to test.

```suggestion
        content = _generate_multi_issues_file_content(self.args.students, self.args.assignments)
        pathlib.Path(MULTI_ISSUES_FILENAME).write_text(content, encodoing=sys.getdefaultencoding())
```

And then, the only tests you have that actually check that there's an output file are like the ones I suggested for running the entire plugin through RepoBee.